.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+-   repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
```

Comment:
You should add quotes around the rev for flake8 too

Reply:
Is pinning required? I don't think we pin on CI meaning this would get out of date.

Reply:
I think it might interesting to have the pre-commit config and let the user decide which one they want to use. Pre-commits main advantage is it being quick blocker at the commit stage.